packages/terra-form-select/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+  * Allow voice-over to read index values for options.
```

Comment:
I left a [comment](https://github.com/cerner/terra-core/issues/3325#issuecomment-792947555) on the issue. Is there a web standards document that defines how this interaction should be working?

Reply:
There isn't a great single standard to base off of. The guidance we are going off of so far are composited between the [samples you have referenced](https://github.com/cerner/terra-core/issues/3325#issuecomment-792947555), patterns for related ui patterns like accessible list filtering (even though they aren't a dropdown, navigating a list is still the same), and industry standard products like Microsoft Teams, etc.